nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -361,6 +361,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - The `hardware.pulseaudio` module now sets permission of pulse user home directory to 755 when running in "systemWide" mode. It fixes [issue 114399](https://github.com/NixOS/nixpkgs/issues/114399).
 
+- The `services.networkmanager.extraConfig` option was changed to use the ini type instead of using a multiline string, this is a breaking change.
```

Comment:
Isn't this the type of config we call "settings" now? Any reason we're not using that name here?

Reply:
not really, would probably be correcter thing now since settings is the name we give rfc42 compliant options. I can add a `lib.mkRemovedOptionModule` error for extraConfig and rename it to settings.

Reply:
fixed